pkg/cmd/attestation/api/client.go

```diff
@@ -63,7 +89,21 @@ func (c *LiveClient) BuildOwnerAndDigestURL(owner, digest string) string {
 // GetByOwnerAndDigest fetches attestation by owner and digest
 func (c *LiveClient) GetByOwnerAndDigest(owner, digest string, limit int) ([]*Attestation, error) {
 	url := c.BuildOwnerAndDigestURL(owner, digest)
-	return c.getAttestations(url, owner, digest, limit)
+	attestations, err := c.getAttestations(url, owner, digest, limit)
```

Comment:
thinking out loud, is there s a good reason for these to be separate functions? (GetByOwner vs GetByRepo) Could the build URL func not just accept empty strings?

Reply:
That's a good point, it might be easy just to the update the build URL func to handle either case. I could take a look in a follow up if that sounds good.

Reply:
Thinking about this again a few days later, I feel like we ought to fetchBundleFromAttestations inside `getAttestations` and not inside these funcs. Let's do that in a follow up.